### PR TITLE
[Docs] Header and search text improvements

### PR DESCRIPTION
**Pull request by tynandebold**
@sampi @zdlm @tynandebold

This pull is directly related to #674 

screen shot 2018-11-02 at 13 18 04

### CODE REVIEW

File: docs/src/less/dox-chrome.less [hunk 1]
```diff
@@ -3,10 +3,10 @@
 .ts-doxchrome {
 	#chrome {
 		.ts-mixin-maximize();
-		top: 30px;
+		top: 40px;
```
Comment (zdlm) [inline docs/src/less/dox-chrome.less]: Can we use @ts-unit-double?

File: docs/src/less/dox-chrome.less [hunk 2]
```diff
@@ -27,8 +27,8 @@
 		z-index: 0 !important;
 		.ts-toolbar-title label {
 			-webkit-font-smoothing: antialiased;
-			font-family: Tradeshifticons;
-			font-size: @ts-fontsize-xxbig + 1px;
+			// font-family: Tradeshifticons;
```
Comment (zdlm) [inline docs/src/less/dox-chrome.less]: delete it, when you don't want to use it anymore.

File: docs/src/less/dox-chrome.less (hunk 1 above)
Comment (tynandebold) [reply docs/src/less/dox-chrome.less]: At this moment, no we can't. It needs to be 40px, if we did double it'll be 44px. We could switch it when we release v11 though.

File: docs/src/less/dox-chrome.less (hunk 2 above)
Comment (tynandebold) [reply docs/src/less/dox-chrome.less]: Removed.

**Pull request merged**